### PR TITLE
Update webpack

### PR DESCRIPTION
**Pull request by jasongrout**
This updates webpack and related plugins, bringing in about 10 months of bugfixes, etc.

### CODE REVIEW

File: tests/test-csvviewer/src/model.spec.ts [hunk 1]
```diff
@@ -9,67 +9,67 @@ import { DSVModel } from '@jupyterlab/csvviewer';
...
     'simple_crlf',
-    require('csv-spectrum/csvs/simple_crlf.csv'),
+    require('csv-spectrum/csvs/simple_crlf.csv').default,
     require('csv-spectrum/json/simple_crlf.json')
   ],
 
   [
     'utf8',
-    require('csv-spectrum/csvs/utf8.csv'),
+    require('csv-spectrum/csvs/utf8.csv').default,
```
Comment (ian-r-rose) [inline tests/test-csvviewer/src/model.spec.ts]: Is there documentation somewhere for these changes @jasongrout ?

Comment (ian-r-rose) [reply tests/test-csvviewer/src/model.spec.ts]: Just read about it on the raw-loader repo. Unfortunate that this breaking change isn't really documented.

Comment (ian-r-rose) [reply tests/test-csvviewer/src/model.spec.ts]: Note for the changelog: this is a breaking change for extension authors.

**Comment by jasongrout** [comment]
The raw loader changes are in its change log https://github.com/webpack-contrib/raw-loader/blob/master/CHANGELOG.md

**Pull request merged**